bin/mge_integrator.py

```diff
@@ -1,9 +1,31 @@
 #!/usr/bin/env python
-
-
-##### This script integrates the results for the Mobilome Annotation Pipeline
-##### Alejandra Escobar, EMBL-EBI
-##### January 11, 2023
+# -*- coding: utf-8 -*-
+
+# Copyright 2025 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os.path
+from map_tools import cds_locator
```

Comment:
can you do
`from map_tools import cds_locator, genomad_parser, ... etc` ?

Reply:
Yes, but I find it cleaner and tidier this way. Then `black` will split that line anyways :D